src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -181,29 +185,29 @@ export const Permission = (permissionProps?: IPermissionProps): JSX.Element => {
   }
 
   const consentTypeProperty = (consented: boolean, item: any): JSX.Element => {
-    if(scopes && scopes.data.tenantWidePermissionsGrant && scopes.data.tenantWidePermissionsGrant.length > 0
-             && consented) {
+    if (scopes && scopes.data.tenantWidePermissionsGrant && scopes.data.tenantWidePermissionsGrant.length > 0
+      && consented) {
 
-      const tenantWideGrant : IPermissionGrant[] = scopes.data.tenantWidePermissionsGrant;
+      const tenantWideGrant: IPermissionGrant[] = scopes.data.tenantWidePermissionsGrant;
       const allPrincipalPermissions = getAllPrincipalPermissions(tenantWideGrant);
       const permissionInAllPrincipal = allPrincipalPermissions.some((permission: string) =>
         item.value === permission);
       return permissionConsentTypeLabel(permissionInAllPrincipal);
     }
-    return <div/>
+    return <div />
   }
 
-  const permissionConsentTypeLabel = (permissionInAllPrincipal : boolean) : JSX.Element => {
-    if(permissionInAllPrincipal){
+  const permissionConsentTypeLabel = (permissionInAllPrincipal: boolean): JSX.Element => {
+    if (permissionInAllPrincipal) {
       return (
-        <div style={{textAlign: 'left', paddingLeft: '10px'}}>
+        <div style={{ textAlign: 'left', paddingLeft: '10px' }}>
```

Comment:
Maybe use `textAlign: 'start` in case the user is working with RTL languages like Arabic.

Reply:
It may also be better to have the padding in a style sheet so changing it is easier.

Reply:
The styles have been addressed in a separate PR that was addressing styling on the Modify Permissions tab: https://github.com/microsoftgraph/microsoft-graph-explorer-v4/pull/2445